exercises/practice/armstrong-numbers/.meta/example.zig

```diff
@@ -0,0 +1,30 @@
+const std = @import("std");
+const math = std.math;
+
+/// Returns the number of digits in `n`.
+/// Uses only integer comparison (no repeated divison, nor log10).
```

Comment:
I'm curious why not log10. Especially if the hardware supports logarithm instructions, I think it might be more optimal (and [easier to quickly understand](https://probablydance.com/2021/07/29/a-logarithm-is-just-the-number-of-digits/)?) than multiplication

If I was mentoring, I'd personally also accept this, and maybe also prefer it:

```
return 1 + @floatToInt(u64, @trunc(@log10(@as(f64, num))));
```

What if this was in a comment as an alternative?